webapp/channels/src/components/announcement_bar/default_announcement_bar/announcement_bar.tsx

```diff
@@ -34,7 +31,6 @@ type Props = {
     modalButtonDefaultText?: string;
     showLinkAsButton: boolean;
     icon?: ReactNode;
-    warnMetricStatus?: WarnMetricStatus;
```

Comment:
also remove references to `AnnouncementBarTypes.ADVISOR` and `AnnouncementBarTypes.ADVISOR_ACK` below and `this.props.warnMetricStatus`?